cumulusci/tasks/github.py

```diff
@@ -207,3 +210,51 @@ def _run_task(self):
                 )
 
                 self.logger.info('Merge conflict on branch {}: created pull request #{}'.format(branch.name, pull.number))
+
+
+class CommitApexDocs(BaseGithubTask):
+
+    task_options = {
+        'branch': {
+            'description': 'branch name',
+            'required': True,
+        },
+        'dir_local': {
+            'description': 'local dir of ApexDocs (contains index.html)',
+            'required': True,
+        },
+        'dir_repo': {
```

Comment:
I named it this way because I wanted to make it clear that the dir on local machine would be copied exactly into the dir at the remote repo. How about `dir_remote`?